pygeohash/geohash.py

```diff
@@ -19,20 +19,20 @@
 <http://www.gnu.org/licenses/>.
 
 Modified 2015 by Will McGinnis for pygeohash.
+Modified 2023 by Paarth Shah for pygeohash.
 """
 
 from math import log10
+from typing import Dict, Tuple
 
 #  Note: the alphabet in geohash differs from the common base32 alphabet described in IETF's RFC 4648
 # (http://tools.ietf.org/html/rfc4648)
 
 __base32 = '0123456789bcdefghjkmnpqrstuvwxyz'
-__decodemap = dict()
-for i in range(len(__base32)):
-    __decodemap[__base32[i]] = i
+__decodemap: Dict[str, int] = {base32_char: i for i, base32_char in enumerate(__base32)}
 
 
-def decode_exactly(geohash):
+def decode_exactly(geohash: str) -> Tuple[float, float, float, float]:
```

Comment:
In theory, I'd like to define a `NamedTuple` for this return value, but I'm not sure how useful it'd be downstream